### ISSUE TITLE
Добавить опциональный фильтр отображения ошибок по автору последнего коммита в строке кода (blame)

### ISSUE THREAD

**Issue by FranchukAndrei** [1C-Company/1c-edt-issues#1606]
### Описание проблемы

Часто, какой-то общий модуль, или модуль формы какой-то сложной обработки разрабатывает несколько авторов. Если модуль содержит большое количество накопленных ошибок, то для навигации по своим/чужим ошибкам, можно включить blame-режим (отображение информации о последнем коммите данной строки или участка кода). Тогда можно пропускать ошибки, которые привнес не я, и сосредоточиться на своих.
Однако это удобно только когда навигация происходит непосредственно в тексте модуля. Навигация по панели ошибок конфигурации в таком режиме неудобна, т.к. она не содержит ошибок об авторе текущей редакции кода, который содержит ошибку. А навигацию по ошибкам я хотел бы выполнять именно на этой панели, используя группировку по важности и типу ошибки - одинаковые ошибки исправлять подряд, в каком бы месте модуля они не находились.

### Описание решения проблемы

Предлагаю добавить в панель информацию о редакции кода, содержащего эту ошибку - автора и дату коммита. По этим колонкам хорошо бы иметь возможность сортировки и отбора.

**Comment by adminimusRU** [comment]
Частично задача решается опцией скрытия ошибок из базовой ветки https://its.1c.ru/db/edtdoc#content:187:hdoc. Но она не скрывает ошибки в метаданных, и не поможет когда ветки разных разработчиков уже влиты в общую.

**Comment by FranchukAndrei** [comment]
> Частично задача решается опцией скрытия ошибок из базовой ветки https://its.1c.ru/db/edtdoc#content:187:hdoc. Но она не скрывает ошибки в метаданных, и не поможет когда ветки разных разработчиков уже влиты в общую.

Да, это пожелание возникло как раз после того, как я пощупал эту опцию в 2024.1.2. Очень полезная опция, как по мне. Но она помогает не привносить новых ошибок в проект, когда работаешь в feature-ветке. А то что предлагаю я, поможет разбираться с теми ошибками, которые уже есть в master, разбирать техдолг.
